From 05217e0cb7ba928392f384cfe1dfe1d64dd8a11a Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Thu, 26 May 2016 11:26:44 -0700
Subject: [PATCH 1583/2748] Revert "YARN-4916. TestNMProxy.tesNMProxyRPCRetry
 fails. Contributed by Tibor Kiss."

This reverts commit 7529f09d4eb69d4004473c4bd74c5f461e80a904.

Change-Id: I2607359d3db9fd274e2ef8c91a97321ad21f7f93
---
 .../nodemanager/containermanager/TestNMProxy.java  |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/TestNMProxy.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/TestNMProxy.java
index 2a6fbdd..937fcbf 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/TestNMProxy.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/TestNMProxy.java
@@ -189,7 +189,8 @@ public void testNMProxyRPCRetry() throws Exception {
       Assert.fail("should get socket exception");
     } catch (IOException e) {
       // socket exception should be thrown immediately, without RPC retries.
-      Assert.assertTrue(e instanceof java.net.SocketException);
+      Assert.assertTrue(e.toString().
+          contains("Failed on local exception: java.net.SocketException"));
     }
   }
 
-- 
1.7.9.5

